Exercise_2.java

```diff
@@ -1,38 +1,67 @@
-public class StackAsLinkedList { 
+// Time Complexity :
+    //Push: O(1)
+    //Pop: O(n)
+    //peek: O(1)
+// Space Complexity : O(n)

```

Comment:
Good Work! Please try O(1) solution for `pop` operation as well.

Reply:
I don't know how to do that in singly linked list. It can be achieved in doubly linked list. Please help me with solution

Reply:
Please try to push new items and pop items from the `head` of the stack. This will be O(1).

Reply:
Got it thanks, updated the code.